content/en/docs/contribute/style/style-guide.md

```diff
@@ -510,6 +510,8 @@ Use ordered headings to provide a meaningful high-level outline of your content.
 Use pound or hash signs (`#`) for non-blog post content. | Use underlines (`---` or `===`) to designate first-level headings.
 Use sentence case for headings in the page body. For example, **Extend kubectl with plugins** | Use title case for headings in the page body. For example, **Extend Kubectl With Plugins**
 Use title case for the page title in the front matter. For example, `title: Kubernetes API Server Bypass Risks` | Use sentence case for page titles in the front matter. For example, don't use `title: Kubernetes API server bypass risks`
+Place relevant links in the body copy. | Include hyperlinks (`<a href=""></a>`) in headings.
```

Comment:
Does "body copy" mean something in particular here? Shouldn't it be just "body" or "body below the Headings"?
Sorry, saw this as a recent contributor and found it a bit confusing, so maybe others might as well :)

Reply:
[Body copy or body text](https://en.wikipedia.org/wiki/Body_text) refers to regular text. In this case, it refers to a nearby paragraph. I don’t want to prescribe that it be above or below, as it depends on the situation.

Reply:
Ah, didn't know that. That makes sense, thanks for sharing 👍🏻